internal/pkg/keybindings/defaults.go

```diff
@@ -39,6 +39,7 @@ var DefaultKeys = map[string]interface{}{
 	"commandpanelup":      gocui.KeyArrowUp,
 	"commandpanelenter":   gocui.KeyEnter,
 	"filter":              rune('/'),
```

Comment:
I wonder if we should take over this one and make fuzzy filter the default

Reply:
Whichever is default, I quite like the idea of `Alt+/` (or `Shift+/`) as the other option 🤷

Reply:
Fuzzy filter now binds to `/` by default, and the non-fuzzy filter binds to `Alt+/`